assets/generated/api.md

```diff
@@ -8,11 +8,7 @@
 {{- range $s := .Services}}
 {{- range $m := $s.Methods}}
 
-{{if $.CustomHeaderIds}}
-## {{GetText $m.Operation.Summary}} {#{{$m.HeaderID}}}
-{{else}}
-## {{GetText $m.Operation.Summary}}
-{{end}}
+## {{GetText $m.Operation.Summary}} {{CustomHeaderId $m.HeaderID}}
```

Comment:
will it add a trailing space if `CustomHeaderId` returns empty string? Compare to current output?

Reply:
it will, but `gofmt` removes it again so the tests work